cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -98,13 +103,23 @@
      */
     Map<String, String> localeMappings;
 
+    /**
+     * Map of locale and the boolean value for fully translated status of the
+     * locale
+     */
+    Map<String, Boolean> localeFullyTranslated = new HashMap<>();
+
     @Override
     public void execute() throws CommandException {
 
         consoleWriter.newLine().a("Pull localized asset from repository: ").fg(Color.CYAN).a(repositoryParam).println(2);
 
         repository = commandHelper.findRepositoryByName(repositoryParam);
 
+        if (onlyIfFullyTranslated) {
+            checkFullyTranslated(repository);
```

Comment:
rename to `initLocaleFullyTranslatedMap`?

Reply:
Done